command-expansion-server/src/app.ts

```diff
@@ -70,8 +71,15 @@ app.put("/expansion-set", async (req, res) => {
   return;
 });
 
-app.get("/command-types/:dictionaryId(\\d+)", async (req, res) => {
-  res.status(501).send("GET /command-types: Not implemented");
+app.post("/command-typescript", async (req, res) => {
+  const commandDictionaryId: string = req.body.input.commandDictionaryId;
+  const commandTypescript = await getCommandTypescriptTypes(db, parseInt(commandDictionaryId, 10));
+  const commandTypescriptBase64 = Buffer.from(commandTypescript).toString("base64");
```

Comment:
Not going to block this PR on this - but worth revisiting later whether we prefer base64 to JSON.stringify here

Reply:
I prefer the base64 option because its more simple than json.stringify (not changing actual bytestream) and is probably more widely available than JSON.stringify

Reply:
I agree with Dylan. base64 is easier to work with. So far Upload, activity typescript generation, and command typescript library are being converted to base64 when being transferred through Hasura actions.